extensions/replication/queueProcessor/task.js

```diff
@@ -35,18 +35,14 @@ metricsProducer.setupProducer(err => {
             log.info('management init done');
 
             const replicationEndpoints = config.getReplicationEndpoints();
-            const bootstrapList = replicationEndpoints.bootstrapList
-                .filter(item => item.site === site);
```

Comment:
I see there is no site assigned to the queue processor. Will this work for one-to-many replication? Just wondering about the intention here.

Reply:
In Orbit we didn't want to spawn a queue_processor process for each single site. With this change we should be able to use a unique queue_processor for any sites.

Reply:
Okay, as long as there is no need to do one-to-many replication that should be fine. Otherwise it will only replicate one of the destinations, I believe.